.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```

Comment:
It's better to make it 120 lines. I don't think the community is ready yet for 160 characters.
Also I would go with newline at the end of all files.